api/filter/fault.proto

```diff
@@ -18,13 +18,17 @@ message FaultDelay {
   }
 
   // Delay type to use (fixed|exponential|..). Currently, only fixed delay (step function) is supported.
-  FaultDelayType type = 1;
+  FaultDelayType type = 1 [(validate.rules).enum.defined_only = true];
 
   // An integer between 0-100 indicating the percentage of operations/connection requests
   // on which the delay will be injected.
-  uint32 percent = 2 [(validate.rules).uint32.lte = 100];
+  uint32 percent = 2 [(validate.rules).uint32 = {
+      required: true,
```

Comment:
I don't think this syntax is quite right, see the CI failures.

Reply:
oh that blows. No required in uint32.. wth.